src/plugins/shared_ux/public/components/solution_toolbar/items/primary_button.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { SolutionToolbarButton, Props as SolutionToolbarButtonProps } from './button';
+
+import './primary_button.scss';
+
+export interface Props extends Omit<SolutionToolbarButtonProps, 'primary'> {
+  isDarkModeEnabled?: boolean;
+}
+
+export const PrimaryActionButton = ({ isDarkModeEnabled, ...props }: Props) => (
+  <SolutionToolbarButton
+    primary={true}
+    className={`solutionToolbar__primaryButton ${
+      isDarkModeEnabled
+        ? 'solutionToolbar__primaryButton--dark'
```

Comment:
I really want to see all of this SASS gone. :-/

Reply:
Removed in [7fa17de](https://github.com/elastic/kibana/pull/124372/commits/7fa17def3ff02bc2f43e2fe17b4429fd877e15ad) - is this the right approach or should I document it in the #122594?